src/Entity/Event.php

```diff
@@ -32,11 +32,21 @@ class Event
      */
     private $dateStart;
 
+    /**
+     * @ORM\Column(type="time")
+     */
+    private $dateTime;
+
```

Comment:
dateStart should probably be named startDate but what's the dateTime for ?

Reply:
Ok the changes are made ...